client/src/components/requests/RequestedList.tsx

```diff
@@ -0,0 +1,151 @@
+                    <Step>
+                        <Step.Content>
+                            <Step.Title>Submitted</Step.Title>
+                        </Step.Content>
+                    </Step>
+                    <Step active>
+                        <Step.Content>
+                            <Step.Title>Approved</Step.Title>
+                        </Step.Content>
+                    </Step>
+                    <Step disabled>
+                        <Step.Content>
+                            <Step.Title>Ready for
+                                Pickup</Step.Title>
+                        </Step.Content>
+                    </Step>
+                </Step.Group>);
+        } else if (r.status === 'READY_FOR_PICKUP') {
+            steps = (
+                <Step.Group stackable='tablet' size='mini'>
+                    <Step>
+                        <Step.Content>
+                            <Step.Title>Submitted</Step.Title>
+                        </Step.Content>
+                    </Step>
+                    <Step>
+                        <Step.Content>
+                            <Step.Title>Approved</Step.Title>
+                        </Step.Content>
+                    </Step>
+                    <Step active>
+                        <Step.Content>
+                            <Step.Title>Ready for
+                                Pickup</Step.Title>
+                        </Step.Content>
+                    </Step>
+                </Step.Group>);
+        } else if (r.status === 'FULFILLED' && r.item.returnRequired) {
+            steps = (
+                <Step.Group stackable='tablet' size='mini'>
+                    <Step active>
+                        <Step.Content>
+                            <Step.Title>Fulfilled</Step.Title>
+                        </Step.Content>
+                    </Step>
+                    <Step disabled>
+                        <Step.Content>
+                            <Step.Title>Returned</Step.Title>
+                        </Step.Content>
+                    </Step>
+                </Step.Group>);
+        } else if (r.status === 'FULFILLED' && !r.item.returnRequired) {
+            steps = (
+                <Step.Group stackable='tablet' size='mini'>
+                    <Step active>
+                        <Step.Content>
+                            <Step.Title>Fulfilled</Step.Title>
+                        </Step.Content>
+                    </Step>
+                    <Step>
+                        <Step.Content>
+                            <Step.Title>Optional Return</Step.Title>
+                        </Step.Content>
+                    </Step>
+                </Step.Group>);
+        }
+
+        return (<Card.Group>
+            <Card key={index} fluid>
+                <Card.Content>
+                    <Card.Header>
+                        <Label medium circular color="blue">
```

Comment:
Use the `ItemAndQuantity` component under `components/desk` for the requested quantity and item display